.github/workflows/main.yml

```diff
@@ -58,50 +58,3 @@ jobs:
     - name: Test with tox
       run: |
         tox -e ${{ matrix.tox-env }}-${{ matrix.qt-lib }} -- -ra --color=yes
-
-  checks:
-
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python
-      uses: actions/setup-python@v3
-      with:
-        python-version: "3.7"
-    - name: Install tox
-      run: |
-        python -m pip install --upgrade pip
-        pip install tox
-    - name: Linting
-      run: |
-        tox -e linting
-    - name: Docs
```

Comment:
Do we really have a check failing the PR status from RTD, if a PR breaks the docs? Scrolling through the commit status list of this PR, I can't seem to find any?

Reply:
Good catch, I have re-synced the webhook from RTD and is now working again:

![image](https://user-images.githubusercontent.com/1085180/197763913-9ea7306c-0b1a-4615-aaeb-b74cfe940980.png)